h2o-py/h2o/explanation/_explain.py

```diff
@@ -1302,6 +1347,9 @@ def ice_plot(
     if frame.type(column) == "string":
         raise ValueError("String columns are not supported!")
 
+    if grouping_variable != None:
```

Comment:
nit: Use `is not` when comparing to `None` rather than equality sign. TL;DR It's possible that object equals `None` even though it's not really `None`. The `is` operator confirms that the variable is actually an instance of `None`, which is always just single object.